### ISSUE TITLE
Discussion: Loss function

### ISSUE THREAD

**Issue by donovanr** [AllenCellModeling/pytorch_integrated_cell#77]
## Issue summary
Talk about how loss function can affect results

## Details
> It would also be nice in the discussion to offer some thoughts and speculations about other ways to tailor the loss function for specific cell biological applications. Rory had some interesting suggestions along these lines when he spoke to us on Friday.

## TODO
- [ ] manuscript text

**Comment by donovanr** [comment]
@gregjohnso you have the most direct experience here -- maybe add some text around MSE vs BCE, how the choice affect results and interoperability?  I personally don't want to go to hard on MSE -> gaussian stuff but anything around how different losses would be better or worse for different structure patterns, BCE good for segmentations, etc might be good to add.

**Comment by gregjohnso** [comment]
see: \subsubsection{Results changing with image normalization could be a big deal}
